app/models/foreman_openscap/arf_report.rb

```diff
@@ -1,8 +1,13 @@
 require 'foreman_openscap/helper'
 
 module ForemanOpenscap
-  class ArfReport < ::Report
-    include Taxonomix
+  class ArfReport < ApplicationRecord
+    LOG_LEVELS = %w[debug info notice warning err alert emerg crit]
+    include ::Foreman::Controller::AvoidLoneTaxonomies
+    include ::Authorizable
+    include ConfigurationStatusScopedSearch
+    include ::Host::Hostmix
+    include ::Taxonomix
```

Comment:
Why is Taxonomix needed here? iiuc reports should only be taxable via the hosts, not directly?